src/oadp_resources/storage_class.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+
+from ocp_resources.resource import Resource
```

Comment:
Can you please create a storage folder and place it there?

Reply:
OKAY Amos